imgs/helmsman-aws/default.nix

```diff
@@ -0,0 +1,36 @@
+{ system ? builtins.currentSystem }:
+
+let
+  nwi = import ../../nwi.nix;
+  pkgs = import ../../pin { snapshot = "nixpkgs-unstable_0"; };
+  callPackage = pkgs.lib.callPackageWith (pkgs // self);
+
+  self = {
+      helm-diff = callPackage ./pkgs/helm-diff { };
+  };
+
+  lib = pkgs.lib;
+  contents = [ pkgs.coreutils pkgs.bash self.helm-diff pkgs.helmsman pkgs.kubernetes-helm pkgs.kubectl pkgs.jq pkgs.awscli ];
+in pkgs.dockerTools.buildImage {
+  inherit contents;
+  name = "nebulaworks/helmsman-aws";
+  # Doesnt matter will use the derivation
+  # when publishing to registry
+  tag = "latest";
+  extraCommands = ''
+      # make sure /tmp exists
+      mkdir -m 1777 tmp
+  '';
+  config = {
+    Env = [ 
+      "PATH=/bin/"
+      "HELM_PLUGINS=/share/helm/plugins/"
```

Comment:
`shellhook` didn't work, using this for now; I will investigate `shellhook` and `makeWrapper` when time permits